hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -210,6 +210,13 @@ module top_${top["name"]} #(
   ${lib.im_defname(sig)} ${lib.bitarray(sig["width"],1)} ${sig["signame"]};
 % endfor
 
+## Inter-module signal collection
```

Comment:
@vogelpi 
thanks! i'm thinking maybe i should template this whole thing based on which clocks clkmgr expects to have an idle signal..what do you think?

Reply:
This would be perfect but depending on the number of use cases it's maybe a bit overkill. For me, it would be fine to do this manually for the moment. Or do you expect every crypto accelerator to have such an idle indicator signal?